components/frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "quality-time-app",
   "version": "4.2.0-rc.5",
   "private": true,
-  "proxy": "http://localhost:5001",
+  "proxy": "http://127.0.0.1:5001",
```

Comment:
Why is this needed? Seems unrelated to the current story?

Reply:
This was needed for me to locally run the frontend for some reason. To be honest I thought I had it put back to localhost already, but will do so now